.bit/responses/4.1-Week 4 Step 1.md

```diff
@@ -100,13 +100,26 @@ To learn more, read the docs [here](https://www.w3schools.com/jquery/html_text.a
 
 </details>
 
-## How do I call the event handler from my HTML form?
+## How do I trigger the function when the form submits?
 
-The `onclick` event executes a Javascript function when a button is clicked
+The event listener triggers the JavaScript function when a button is clicked.
+Here is how the form should look in the bunnimage index.html file:
 
 ```html
-  <label for="name">Code: </label> <br>
-  <input type="text" id="username" name="name" aria-label="name" /> <br>
-  <input value="Submit" type="button" onclick="getImage()" />
+<form id="bunnForm">
+   <label for="name">Code: </label> <br>
+   <input type="text" id="username" name="name" aria-label="name" /> <br>
+   <input value="Submit" type="button" />
+</form>
+```
+
+Here is the JavaScript:
+
+```JavaScript
+const bunnForm = document.getElementById('bunnForm');
+
+bunnForm.addEventListener('submit', function (event) {
+   $('#output').text(document.getElementById("username").value + "❤️")
```

Comment:
I know this has not changed but I wonder is using JQuery is really a good idea, this can be done in plain JS using textContent https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent

Reply:
@blueboxes Do you think there is any benefit in introducing jQuery at all?

Reply:
@emsesc @blueboxes I always thought that jQuery had fallen out of favor

Reply:
@ganning127 @lazyplatypus any thoughts?

Reply:
jQuery was designed to solve a problem that all browsers worked differently and were not everygreen updated. This is no longer the case and everything in jQuery can be done in vanilla js. 

There is a modern lightweight jQuery that does not support the weird old browsers but this is pointless. So yes in short I would not intro jQuery.

Reply:
i would just use vanilla js to do this with either .text or .innerHTML

Reply:
I also thought that innerHTML presented a security risk. Anyone know?